### PR TITLE
resize mode finishing up

### PR DESCRIPTION
**Pull request by arekkubaczkowski**
- [x] Testing
- [x] Automated tests need to be written
- [x]  Doc updates
- [x] Add the resizeMode functionality to the example apps so that customers can try it and our manual QA team can trigger the different resizeModes

QA Criteria: 
 - Playground Example screen presents the video with according configuration options
 - Settings can be found once press the top right corner icon
 - Settings modal presents the list of available configurations along with the `resizeMode` option
 - SDK offers 3 kinds of resize modes: `Aspect Zoom`, `Aspect Fit` and `Aspect Fill` (https://github.com/arekkubaczkowski/amazon-ivs-react-native-player/blob/feat/resize-mode-finishing-up/docs/ivs-player-reference.md#resizemode-optional)
 - Selected option should be applied immediately and visible on video player

https://user-images.githubusercontent.com/18654447/155139507-928e5383-864c-4cb3-a8f4-713e7332a18e.mov

### CODE REVIEW

**Comment by alexmdodge** [comment]
@arekkubaczkowski The feature branch has been updated with the original contributors code, good to fix the merge conflicts here and perform final testing 👍

**Comment by RupamShaw** [comment]
I tried with android its not resizing for aspectFill

**Comment by RupamShaw** [comment]
i thought like ios if I hard code in android then no need to set in [src/IVSPlayer.tsx] but once i setup this file too all good working

**Comment by RupamShaw** [comment]
Yeah cant wait release this  resize mode to get release

File: example/src/screens/PlaygroundExample.tsx [hunk 1]
```diff
@@ -128,7 +154,7 @@ export default function PlaygroundExample() {
           initialBufferDuration={initialBufferDuration}
           playbackRate={playbackRate}
           progressInterval={progressInterval}
-          volume={volume}
+          volume={0}
```
Comment (alexmdodge) [inline example/src/screens/PlaygroundExample.tsx:157]: Was this added for debugging? Assuming we should revert to the variable?

**Review by alexmdodge — CHANGES_REQUESTED** [review]
Looks good 👍 One small change which may have been leftover from debugging.

File: example/src/screens/PlaygroundExample.tsx (hunk 1 above)
Comment (alexmdodge) [reply example/src/screens/PlaygroundExample.tsx:157]: This is a small tweak that I can make on the final feature branch merge 👍

**Review by alexmdodge — APPROVED** [review]

**Pull request merged**